components/function-runtimes/README.md

```diff
@@ -0,0 +1,5 @@
+# Function Runtimes
+
+## Overview
+
+This project includes Docker images, which are used as a base on which serverless Functions are build. See [this](../../docs/serverless) directory for more documentation for serverless Functions.
```

Comment:
```suggestion
This project includes Docker images which are used as a base for serverless Functions. See the [`docs/serverless`](../../docs/serverless) directory for more documentation on serverless Functions.
```
The comma after `which` must be removed, and `for` should be changed to `on`. Other changes are optional, but I would recommend them.

Reply:
Seems reasonable :D